gatsby/src/components/introItem.js

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import * as styles from "./introItem.module.css";
+import { useStaticQuery, graphql } from "gatsby";
```

Comment:
style suggestion: I suggest you put the `import` from third-party library first, then the `import` from the local folder. So that they are more grouped together. Some people will also add a blank line between these two parts.

In this case:
```
import * as React from "react";
import { useStaticQuery, graphql } from "gatsby";

import * as styles from "./introItem.module.css";